### PR TITLE
Display a rotating search-query example

### PR DESCRIPTION
**Pull request by gjvis**

### CODE REVIEW

**Comment by jessstubenbord** [comment]
retail
cook
administration
driving
sales
nurse
school
communication
airport

**Comment by gjvis** [comment]
ping @lukasz-kaniowski @nathancashmoredwp @jessstubenbord this is now ready to go

File: app/views/searches/_form.html.erb [hunk 1]
```diff
@@ -2,6 +2,9 @@
   <div class="form-group">
     <label class="form-label" for="query">
       <span class="heading-small">Roles, skills and interests</span>
+      <% if local_assigns[:show_hints] %>
+        <span class="form-hint">e.g. <span class="example">Retail</span></span>
```
Comment (nathancashmoredwp) [inline app/views/searches/_form.html.erb]: class js-hidden will load in a more graceful manor when you first hit the page.  Can display the full html tag in some cases when JS turned off.

**Comment by lukasz-kaniowski** [comment]
:shipit:

**Pull request merged**